.github/workflows/ou3-p3-accelerate-canonical-once.yml

```
@@ -0,0 +1,193 @@
+          text = text.replace(
+              'def _certified_delta(Pz, h: float, upper: list[float]) -> float:',
+              'def _certified_delta_bisection(Pz, h: float, upper: list[float]) -> float:',
+              1,
+          )
+
+          marker = '\n\n@functools.lru_cache(maxsize=4)\ndef common_boundary_floor'
+          if marker not in text:
+              raise SystemExit('cannot find common-boundary insertion point')
+          insert = r'''
+
+def _validated_inverse(Pz):
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected key

(syntax-check)